README.md

```diff
@@ -20,6 +20,23 @@ The Apollo Client Browser Devtools appear as an "Apollo" tab in your web browser
 
 ![Apollo Client Browser Devtools](./assets/ac-browser-devtools-3.png)
 
+## Apollo Maintainers
```

Comment:
```suggestion
## Maintainers
```

Can we drop the "Apollo" here?  I think one interpretation could be that we're the "Apollo Maintainers" for the company and I want to make sure we're setting expectations accordingly